app/posts/manage-teacher-training-applications/2021-02-25-filter-by-subject.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Filtering applications by subject
+description: Let users filter the applications list by the course’s subject
+date: 2021-02-25
+---
+
+Users need to be able to filter their applications by subject.
```

Comment:
I think it would be good to say a little more about this need, even if it's just that we've seen it in research.

Reply:
Agree, I think we can get some details from @billwessel-dfe